.travis.yml

```diff
@@ -5,8 +5,19 @@ python:
   - "3.6"
   - "3.7"
 # command to install dependencies
+addons:
+  apt:
+    packages:
+      - clamav-daemon
 install:
   - pip install tox-travis coveralls
+before_script:
+  # we need to download the clamav signature database before the daemon will run
+  - sudo service clamav-freshclam stop
+  - sudo freshclam
+  - sudo service clamav-daemon restart
+  - sudo service clamav-daemon status
+  - while ! test -S /var/run/clamav/clamd.ctl; do printf "."; sleep 1; done
```

Comment:
maybe if we wrap this all up in a single shell command it would be faster? (why) do we need to restart the daemon to fetch new entries?

Reply:
I don't understand your comment about wrapping this up in a single shell command for speed.

We need to (re)start the daemon _after_ fetching the virus signature database because otherwise it won't be runnning!  The systemd service definition for clamav-daemon has

```
# Check for database existence
ConditionPathExistsGlob=/var/lib/clamav/main.{c[vl]d,inc}
ConditionPathExistsGlob=/var/lib/clamav/daily.{c[vl]d,inc}
```

which tells it to not start up if there virus DB is not present.

(I thought I commented about this somewhere on GitHub, but I can't find my comment.  Maybe it got lost; I had some disappearing public wifi problems during lunch today.)

Reply:
> I don't understand your comment about wrapping this up in a single shell command for speed.

i doubt it's a real factor, but i figured that each line in that script might be a different call. that could be my `Dockerfile` paradigm leaking though. ;) i don't know how travis works, it's just a silly assertion.

> We need to (re)start the daemon after fetching the virus signature database because otherwise it won't be runnning! The systemd service definition for clamav-daemon has
> [...]

makes sense. i guess i was hoping to shave of some seconds here by avoiding a needless restart, but if it's not running at all, i guess we need to bite the bullet anyways.

thanks for your work!